tests/integration_test.rs

```diff
@@ -321,24 +335,39 @@ fn sha256_digest_bytes(path: &PathBuf) -> String {
 }
 
 #[rstest]
-#[case(Platform::Linux64)]
-#[case(Platform::LinuxAarch64)]
-#[case(Platform::LinuxPpc64le)]
-#[case(Platform::OsxArm64)]
-#[case(Platform::Osx64)]
-#[case(Platform::Win64)]
+#[case(Platform::Linux64, false)]
+#[case(Platform::LinuxAarch64, false)]
+#[case(Platform::LinuxPpc64le, false)]
+#[case(Platform::OsxArm64, false)]
+#[case(Platform::Osx64, false)]
+#[case(Platform::Win64, false)]
+#[case(Platform::Linux64, true)]
+#[case(Platform::LinuxAarch64, true)]
+#[case(Platform::LinuxPpc64le, true)]
+#[case(Platform::OsxArm64, true)]
+#[case(Platform::Osx64, true)]
+#[case(Platform::Win64, true)]
```

Comment:
```suggestion
#[case(Platform::Linux64, false)]
#[case(Platform::Linux64, true)]
#[case(Platform::LinuxAarch64, false)]
#[case(Platform::LinuxAarch64, true)]
#[case(Platform::LinuxPpc64le, false)]
#[case(Platform::LinuxPpc64le, true)]
#[case(Platform::OsxArm64, false)]
#[case(Platform::OsxArm64, true)]
#[case(Platform::Osx64, false)]
#[case(Platform::Osx64, true)]
#[case(Platform::Win64, false)]
#[case(Platform::Win64, true)]
```

sorting these by platform first and then use_pypi is a bit more readable imo